shardingsphere-kernel/shardingsphere-transaction/shardingsphere-transaction-core/src/main/java/org/apache/shardingsphere/transaction/spi/TransactionConfigurationFileGenerator.java

```diff
@@ -41,9 +41,10 @@
     /**
      * Get transaction configuration.
      *
-     * @param transactionRuleConfiguration transaction rule configuration
+     * @param originTransactionProps transaction rule props
```

Comment:
The description of param `originTransactionProps  ` seems to not match very much